cpu/lpc11u34/periph/spi.c

```diff
@@ -80,6 +80,7 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
         case SPI_SPEED_1MHZ:
             spi->CR0 |= (11 << 8);
             break;
+        case SPI_SPEED_4MHZ:
```

Comment:
This makes no sense to me. Either support the mode or not. If the setup under `SPI_SPEED_5MHZ` leads to 4MHz one should place the code to the right option and not support the other